component/src/main/java/org/cbioportal/genome_nexus/component/annotation/RefSeqResolver.java

```diff
@@ -22,13 +23,35 @@ public RefSeqResolver(CanonicalTranscriptResolver canonicalTranscriptResolver)
     @Nullable
     public String resolve(VariantAnnotation variantAnnotation)
     {
-        return this.resolve(this.canonicalTranscriptResolver.resolve(variantAnnotation));
+        return this.resolve(variantAnnotation, null);
+    }
```

Comment:
@onursumer should we keep these `resolve` functions without a predefined list? To me it seems like we are mostly keeping them at the moment to support our test cases. I don't think there's an actual use case for using resolve without the predefined list. Is that right?

Reply:
Right, currently we only use these `resolve` functions in tests. We can update the tests and remove the unused functions  if you think there is no actual use case for them.

Reply:
yeah I don't think there's a use case, so best to remove them to avoid people from accidentally using them. But could be in follow up